actions/check-thirdparty-notice/action.yml

```diff
@@ -0,0 +1,84 @@
+# SPDX-FileCopyrightText: Copyright (C) 2025 Opal Health Informatics Group at the Research Institute of the McGill University Health Centre <john.kildea@mcgill.ca>
+#
+# SPDX-License-Identifier: TBD
```

Comment:
@mschoettle TBD, for the two new files

Reply:
Sounds good. I'd seen the license file, but since it said "unless otherwise stated", I wanted to make sure MIT was what we wanted. Updated.